app/views/bloggers/index.html.haml

```diff
@@ -1,8 +1,9 @@
 - provide(:title, 'User Blogs')
 .generic-container
-- if !@blog.empty?
+- if !@blogs.empty?
```

Comment:
negation is discouraged in programming avoid ! where ever is possible
why not we use if @blogs.present? or @blogs.exist?